subprojects/docs/src/main/resources/header.html

```diff
@@ -225,6 +225,7 @@ <h3 id="authoring-gradle-builds-java">Authoring JVM Builds</h3>
             <li><a href="../userguide/dependency_management_for_java_projects.html">Managing Dependencies</a></li>
             <li><a class="nav-dropdown" data-toggle="collapse" href="#jvm-plugins" aria-expanded="false" aria-controls="jvm-plugins">JVM Plugins</a>
                 <ul id="jvm-plugins">
+                    <li><a href="../userguide/java_plugin.html">Java Plugin</a></li>
```

Comment:
I wouldn't include this change in the PR. I wonder if there is a reason not to include the Java plugin in this section. For example, you shouldn't apply the `java` plugin any more, you should only apply the more specialized plugins (I think).

Reply:
Removed